frontend/src/data/saga.ts

```diff
@@ -1,16 +1,16 @@
 import { all, call, put, takeLatest } from "redux-saga/effects";
 import FirebaseInteractor from "../firebase/firebaseInteractor";
+import { Action, ActionTypes, CreateUserParams, LoginParams, Word } from "../models/types";
 import {
   authenticationRequest,
   getWordsRequest,
-  singleRequest,
+  singleRequest
 } from "./actions";
-import {Action, ActionTypes, LoginParams, Word} from "../models/types";
 
 let firebaseInteractor = new FirebaseInteractor();
 
 export default function* rootSaga() {
-  yield all([root()]);
+  yield all([root(), waitForSignInAtStart()]);
 }
 
 function* root() {
```

Comment:
can we get rid of this it seemed like we don't rly need singlerequest anymore. Also can you delete the one in action too?

Reply:
What do you want to get rid of? I am not sure which lines you are referring to.

Reply:
Ohhh tag the wrong line... line 26-31 in this file. The singleRequest.success() and singleRequest.error()